pepper-apis/src/main/resources/sql.conf

```diff
@@ -1,79 +1,4 @@
 {
-    "listUserActivitiesQuery": """
```

Comment:
Woohoo! Finally got rid of this query! We're one step closer to ditching this `sql.conf` file!